runtime/src/bank.rs

```diff
@@ -6600,10 +6600,23 @@ impl Bank {
         )
     }
 
+    /// Return the accumulated executed transaction count
     pub fn transaction_count(&self) -> u64 {
         self.transaction_count.load(Relaxed)
     }
 
+    /// Return the transaction count executed only in this bank
+    pub fn executed_transaction_count(&self) -> u64 {
+        let mut executed_transaction_count =
+            self.transaction_count() - self.parent().map_or(0, |parent| parent.transaction_count());
+        if !self.bank_tranaction_count_fix_enabled() {
```

Comment:
This will break master, actually, as I fixed the `tranaction` typo everywhere: https://github.com/solana-labs/solana/pull/29347
Can you please rebase?

Reply:
Done